indico/modules/events/abstracts/controllers/management.py

```diff
@@ -155,3 +174,84 @@ class RHAbstractListStaticURL(RHAbstractListBase):
 
     def _process(self):
         return jsonify(url=self.list_generator.generate_static_url())
+
+
+class RHCreateAbstract(RHAbstractListBase):
+    def _process(self):
+        abstract_form_class = make_abstract_form(self.event_new)
+        form = abstract_form_class(event=self.event_new)
+        if form.validate_on_submit():
+            data = form.data
+            if isinstance(data['submitted_for_tracks'], Track):
+                data['submitted_for_tracks'] = {data['submitted_for_tracks']}
+            elif data['submitted_for_tracks'] is None:
+                data['submitted_for_tracks'] = set()
+            abstract = create_abstract(self.event_new, *get_field_values(data))
+            flash(_("Abstract '{}' created successfully").format(abstract.title), 'success')
+            tpl_components = self.list_generator.render_list(abstract)
+            if tpl_components.get('hide_abstract'):
+                self.list_generator.flash_info_message(abstract)
+            return jsonify_data(**tpl_components)
+        return jsonify_template('events/abstracts/forms/abstract.html', event=self.event_new, form=form)
+
+
+class RHDeleteAbstracts(RHManageAbstractsActionsBase):
+    def _process(self):
+        include_contribs = request.values.get('include_contribs') == '1'
+        contrib_counter = 0
```

Comment:
`num_contribs`? Also, why not something like this?

    num_contribs = sum(1 for a in self.abstracts if a.contribution) if include_contribs else 0